classes/services/PKPSiteService.inc.php

```diff
@@ -0,0 +1,333 @@
+<?php
+/**
+ * @file classes/services/PKPSiteService.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class PKPSiteService
+ * @ingroup services
+ *
+ * @brief Helper class that encapsulates business logic for the overall site
+ */
+
+namespace PKP\Services;
+
+use \Application;
+use \DAORegistry;
+use \ServicesContainer;
+use \PKP\Services\EntityProperties\PKPBaseEntityPropertyService;
+
+class PKPSiteService extends PKPBaseEntityPropertyService {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
```

Comment:
(Remove pointless constructors)

Reply:
I _think_ that these are not pointless, because they pass `$this` into the parent constructor. Kassim set this up so I'm not sure what problem it was solving.

Reply:
Weird, I'm not sure what that pattern is for, but let's leave it for now. Bigger fish to fry.